### PR TITLE
Addresses GH issue #453: Allow single KT to run and not automatically…

### PR DESCRIPTION
**Pull request by ybyzek**
… destroy resources

This PR executes option 1 documented at https://github.com/confluentinc/kafka-tutorials/issues/453#issuecomment-649720525 but I am happy to implement another option if anyone would like to propose for a better one.  Quick summary of PR is ability to run the following to leave the KT running

```
make SEQUENCE="dev, test"
```

Note to self: programmatic way to make file changes:

```
find . -type f -name "Makefile" | xargs perl -pi -e '$_ .= qq(SEQUENCE := "dev, test, prod"\n) if /shell mktemp/' 
find . -type f -name "Makefile" | xargs perl -pi -e 's/\$\(TEMP_DIR\)/\$\(TEMP_DIR\) \$\(SEQUENCE\)/g'
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,30 +252,68 @@ Lastly, create a Makefile in the `code` directory to invoke the harness runner a
...
+```
+(cd _includes/tutorials/transforming/kstreams/code/ && make SEQUENCE="dev, test")
+```
+
+Now you can play with the environment, some sample commands shown below.  Don't forget to shutdown Docker containers when you are done.
+
+```
+docker exec broker kafka-topics --list --bootstrap-server localhost:9092
+docker exec -it ksqldb-cli ksql http://ksqldb-server:8088               
+```
```
Comment (bbejeck) [inline README.md:314]: Now you can play with the environment, some sample commands shown below. 

```
docker exec broker kafka-topics --list --bootstrap-server localhost:9092
docker exec -it ksqldb-cli ksql http://ksqldb-server:8088   
```            
Don't forget to shutdown Docker containers when you are done

```
docker-compose down --volumes 
```

Comment (ybyzek) [reply README.md:314]: @bbejeck `docker-compose ` commands work only when in the same directory as the docker-compose.yml file.  Given that we used paranthesis above, `docker-compose down` won't work.

Even more so, when I go to the `code` directory, `docker-compose down` still doesn't work, even when specifying the file:

```
[13:41:49] ~/git/kafka-tutorials/_includes/tutorials/transforming/kstreams/code(GH-453) ✗: docker-compose -f docker-compose.yml down
Removing network code_default
WARNING: Network code_default not found.
```

What if we just tell users how to get docker container info with:

```
docker container ls
docker container rm <id>
```

Comment (bbejeck) [reply README.md:314]: yep that works for me

Comment (ybyzek) [reply README.md:314]: @bbejeck Unfortunately `docker container ls` does not provide the tutorial name:

```
[14:09:06] ~/git/kafka-tutorials(GH-453) ✗: docker container ls
CONTAINER ID        IMAGE                                    COMMAND                  CREATED             STATUS              PORTS                                        NAMES
7e9fc9ffaf24        confluentinc/ksqldb-server:0.7.1         "/usr/bin/docker/run"    12 seconds ago      Up 11 seconds       0.0.0.0:8088->8088/tcp                       ksqldb-server
f9441e503e6a        confluentinc/cp-schema-registry:5.5.0    "/etc/confluent/dock…"   23 minutes ago      Up 23 minutes       0.0.0.0:8081->8081/tcp                       schema-registry
d0b328c59187        confluentinc/cp-enterprise-kafka:5.5.0   "/etc/confluent/dock…"   23 minutes ago      Up 23 minutes       9092/tcp, 0.0.0.0:29092->29092/tcp           broker
3d321fee7116        confluentinc/cp-zookeeper:5.5.0          "/etc/confluent/dock…"   23 minutes ago      Up 23 minutes       2888/tcp, 0.0.0.0:2181->2181/tcp, 3888/tcp   zookeeper
```

Comment (bbejeck) [reply README.md]: I meant `docker network ls` since it seems the tutorial name is used for the network name.  
At any rate, what you have is fine

**Comment by ybyzek** [comment]
@bbejeck I saw your Slack message about merging this PR, but before I do, I wanted to highlight that one Semaphore test appears to be failing.  I don't believe it's related to this PR, but I would appreciate your review and approval/confirmation to merge.  Thanks in advance.

**Comment by bbejeck** [comment]
@ybyzek test failures are not related to this PR, so feel free to merge.  

Unfortunately, we see some intermittent failures related to #396

**Pull request merged**